src/pages/institution/institution-solution/institution-solution.ts

```diff
@@ -52,9 +52,7 @@ export class InstitutionSolutionPage {
       .getInstituionFromStorage()
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status